sockeye/constants.py

```diff
@@ -334,7 +334,9 @@
                    OUTPUT_HANDLER_ALIGN_TEXT,
                    OUTPUT_HANDLER_BEAM_STORE]
 OUTPUT_HANDLERS_SCORING = [OUTPUT_HANDLER_SCORE,
-                           OUTPUT_HANDLER_PAIR_WITH_SCORE]
+                           OUTPUT_HANDLER_PAIR_WITH_SCORE,
+                           OUTPUT_HANDLER_TRANSLATION_WITH_SCORE,
+                           OUTPUT_HANDLER_TRANSLATION_WITH_ALIGNMENT_MATRIX]
```

Comment:
This is incorrect—the purpose of `OUTPUT_HANDLERS_SCORING` is to list valid values of `--output-type` for `sockeye.score`.